backend-es/test/snapshots/Snapshot.PrimOnLocals.purs

```diff
@@ -0,0 +1,27 @@
+-- @inline Snapshot.PrimOnLocals.a always
+-- @inline Snapshot.PrimOnLocals.b always
+-- @inline Snapshot.PrimOnLocals.c always
+
+module Snapshot.PrimOnLocals where
+
+import Prelude
+
+import Data.Array as Array
+
+a :: forall a. Array a -> Int
+a x = Array.length x + (2 * Array.length x)
+
+b :: forall a. Array a -> Int
+b x = 42 + 55 + Array.length x
+
+c :: forall a. Array a -> Int
+c x = Array.length x
```

Comment:
I think this test could use some editing.

* I don't think `b` and `c` are good tests for this. The array for binding x will trivially inline so it doesn't actually test that it's using the NeutLocal reference.
* I think that `a` would be better if you use the array reference in two different ways. For example, take an array of Ints, and snoc on it's length. By using it in syntactically identical ways, I could see this tripping up against a hypothetical CSE optimization.
* I think that this should be renamed to be specific to what you want to test, like "InlineLocalReferenceOpXX" or